JavaScript/Topics/4.Command_Line/README.md

```diff
@@ -1 +1,75 @@
-# To Do
\ No newline at end of file
+# Introduce JavaScript's usage in the Command-Line 
+<br>
+
+***JavaScript*** is a powerful, widely used, open-source, and dynamic programming language used to add and render interactivity on websites. Also, a variety of other developer tools have been built on top of it, giving rise to a vast range of functionalities that can be implemented with ease to add on the dynamism of the website.<br>
+
+Besides using JS to render user-friendly interactive functionalities and features on our websites, a JavaScript shell (Command-line) enables us to quickly run and test snippets of JS code without having to load and reload it in a web page. This makes developing and debugging JS code extremely easy. These JS program that run on Command Line Interfaces (CLI) are the standalone JS scripts.
+<hr>
+Some JavaScript shells that are stand-alone environments are:
+
+- ***NodeJS*** - Node.js is a platform for easily building fast, scalable network applications.
+- ***GraalJS*** - A high performance implementation of the JavaScript programming language. Built on the GraalVM by Oracle Labs.
+- ***ShellJS*** - Portable Unix shell commands for Node.js
+
+***Node.js*** is a popular JS runtime environment for creating web applications on server side, that is it allows us to run JS on the backend outside the browser. It comes with a lot of built-in modules and it enables us to run and test JS code very fast and efficiently mainly because of its asynchronous event run model running on Google V8 engine.
+<br>
+
+Run our first JS program as follows : <br>
+Having Node.js installed in our system, to run a stand-alone JS program in node.js, we follow steps as :<br>
+- Open any text editor and create a JS file with any piece of JS sample code. Like, here, I used VS code to create the js file with the code shown. 
+
+<image src="https://github.com/TishaJhabak1014/winter-of-contributing/blob/Javascript/JavaScript/Topics/4.Command_Line/divv8.PNG">
+ 
+- Open terminal or command prompt
+- Change the directory and file path to where the JS file is located
+- Run the code `node <name of the file>`
+ 
+<image src="https://github.com/TishaJhabak1014/winter-of-contributing/blob/Javascript/JavaScript/Topics/4.Command_Line/divv7.PNG">
+ 
+- Verify the output.
+<hr>
+ 
+Now, coming to ***NPM***, a popular package manager for JavaScript, can be used to install almost any node modules, we need to work on the front-end of our applications. Similarly, ***NodeJs***, besides backend, can efficiently be used for the front-end of our application. 
```

Comment:
```suggestion
Now, coming to ***NPM***, a popular package manager or package registry for JavaScript commonJS modules, which can be used to packaging almost any commonJS modules and makes it ready for distribution and once your modules are distributed then anyone having node installed in their computers can install almost any any modules and after getting installed these packages get saved into a directory named Node Modules (which contains both package and its dependencies), we need to work on the front-end of our applications. Similarly, ***NodeJs***, besides backend, can efficiently be used for the front-end of our application. 
```
Please check once for grammatical mistakes and resolve yourself and commit

Reply:
also write npm install package

or 

npm install package@version
because there is no example which defines it syntax, i always said to include examples if you claim anything